src/firecracker/src/main.rs

```diff
@@ -86,7 +86,7 @@ impl From<MainError> for ExitCode {
             _ => FcExitCode::GenericError,
         };
 
-        ExitCode::from(exit_code as u8)
+        exit_code
```

Comment:
There's a failing lint: https://buildkite.com/firecracker/firecracker-pr/builds/7720#018c1042-de7e-41c8-8263-f1009f7032b3/55-1005

I think it can be avoided by avoiding the let and just returning the `match`?